tests/unit/test_customers.py

```
@@ -71,9 +110,551 @@ def test_retrieve_customer(self, client: PlaneClient, workspace_slug: str, custo
+        customer = client.customers.create(workspace_slug, data)
+        try:
+            assert customer.name == data.name
+            assert customer.email == data.email
+            assert customer.domain == data.domain
+            assert customer.employees == 42
+            assert customer.revenue == "1000000"
+        finally:
+            try:
+                client.customers.delete(workspace_slug, customer.id)
+            except Exception:
+                pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)